docs/spec.md

```diff
@@ -2363,50 +2379,38 @@ planning to address this in
 
 #### Outputs
 
-| Name     | Type                       | Constraints                    |
-|----------|----------------------------|--------------------------------|
-| `result` | tensor or quantized tensor | (C12-C13), (C15), (C17), (C20) |
+| Name     | Type                       | Constraints                |
+|----------|----------------------------|----------------------------|
+| `result` | tensor or quantized tensor | (C12), (C14), (C16), (C19) |
 
 #### Constraints
 
-<!-- markdownlint-disable line-length -->
```

Comment:
Consider removing the rest of the markdown flags by wrapping expression in a new line?

Reply:
I removed a few more flags, but for some ops line wrapping significantly degrades readability so I didn't touch those.